src/main/kotlin/org/jetbrains/plugins/bsp/magicmetamodel/impl/workspacemodel/WorkspaceModelToModulesMapTransformer.kt

```diff
@@ -103,7 +104,8 @@ public object WorkspaceModelToModulesMapTransformer {
           libraries = libraries.map { PythonLibrary(it.sourceJars) }.toList(),
           sdkInfo = entity.getPythonSdk(),
         )
-      } else {
+      // TODO: add support for Go
```

Comment:
I would move that comment above
`val module = if`
but I can do that when merging all other PRS